### ISSUE TITLE
Header overflows the parent div

### ISSUE THREAD

**Issue by cyberxndr** [bvaughn/react-virtualized#231]
Hi, @bvaughn. There is an issue with FlexTable's header (and maybe with others).  Header overflow appears in case when FlexTable rendered inside a parent with width < 100%.

Please, run the following example to get the details https://cdn.rawgit.com/cyberxndr/778e622610967560c209f820ad70b5ee/raw/97b5a5c9fa19788de6e97c4b3f833070ec5821a0/example.html

`* { box-sizing: border-box }` solves the problem.

**Comment by bvaughn** [comment]
Hey @cyberxndr,

Firstly, thanks for the example code! Unfortunately I don't see the overflow you're describing.

**Comment by cyberxndr** [comment]
Chrome 50.0.2661.94
Firefox 45.0.1
![2016-05-11-182017_scrot](https://cloud.githubusercontent.com/assets/2040537/15186399/8e1ecff2-17a5-11e6-95a8-55b3387a57e0.png)

**Comment by bvaughn** [comment]
Chrome 50.0.2661.94:
<img width="337" alt="screen shot 2016-05-11 at 8 31 44 am" src="https://cloud.githubusercontent.com/assets/29597/15186685/d4622476-1752-11e6-8325-4da1e7ca95ed.png">

**Comment by bvaughn** [comment]
Furthermore, looking at the inline styles for the `FlexTable__headerRow` cell on the demo you linked:
```css
  height: 60px;
  padding-right: 2px;
  width: 300px;
```
That seems right since the parent `container` styles are:
```css
    height: 200px;
    width: 300px;
    display: flex;
    flex: 1;
```

**Comment by babsonmatt** [comment]
For what it's worth (Version 50.0.2661.94 (64-bit)) on OSX:
![image](https://cloud.githubusercontent.com/assets/853220/15186751/4629a458-176c-11e6-8220-3cfdc7b17b7f.png)

**Comment by bvaughn** [comment]
To be clear, I prefer using `* { box-sizing: border-box }` everywhere. I think it should be the default. I'm just not seeing what you're seeing.

**Comment by cyberxndr** [comment]
It was a small surprise for me to get this behaviour. Thanks #103 it solved now. Hope this will helps somebody too.

**Comment by bvaughn** [comment]
I'm glad you were able to find a resolution. I'm still a little confused about why I was unable to reproduce it.

Would you clarify- which component you had to add the `box-sizing` rule to @cyberxndr ? I don't mind adding it to the default react-virtualized styles.

**Comment by cyberxndr** [comment]
@bvaughn Seems like `.FlexTable__headerRow, .FlexTable__Grid { box-sizing: border-box }` is enough.

Single `.FlexTable__headerRow { box-sizing: border-box }`:

![2016-05-11-220117_scrot](https://cloud.githubusercontent.com/assets/2040537/15193177/34188ae6-17c5-11e6-85c5-140a1fd916db.png)

Both `.FlexTable__headerRow, .FlexTable__Grid { box-sizing: border-box }`:

![2016-05-11-220127_scrot](https://cloud.githubusercontent.com/assets/2040537/15193181/399d11bc-17c5-11e6-981e-e6adcf67624c.png)

Also the version with `rowCount=2` and no `box-sizing` rules:

![2016-05-11-221725_scrot](https://cloud.githubusercontent.com/assets/2040537/15193371/49f9b7bc-17c6-11e6-9200-00239494b1a2.png)

Seems like it somehow related to the scroll bar. Header accumulates right padding value in this case. No rules required when there is no scroll bar to display correctly. Have no thoughts why, however.

**Comment by bvaughn** [comment]
Thanks!

Published 7.0.3 with `box-sizing: border-box` rule added to `.FlexTable__headerRow` and `.FlexTable__Grid`.
